mri/test/test_optimizer.py

```diff
@@ -37,8 +32,14 @@ def setUp(self):
         print("[info] Image loaded for test: {0}.".format(
             [im.data.shape for im in self.images]))
         self.mask = get_sample_data("mri-mask").data
-        self.names = ["MallatWaveletTransform79Filters"]
-        print("[info] Found {0} transformations.".format(len(self.names)))
+        # Test a wide variety of linear operators :
+        # From WaveletN
+        self.decimated_wavelets = ['sym8']
+        # From WaveletUD2, tested only for analysis formulation
+        self.undecimated_wavelets = [24]
+        # , 24] TODO debug and add this for testing
```

Comment:
I don't understand this comment

Reply:
Ahh, this was for fixing above described assumed bug for waveletUD. Now it doesn't make sense. Will remove it.